cmd/pbstatuser/main.go

```diff
@@ -41,7 +42,20 @@ type SourceInfo struct {
 	Identity identity.XRHID
 }
 
+const (
+	metricNameSourceAvailabilityCheckRequestTotal = "source_availability_check_request_total"
+)
+
 var (
+	totalAvailabilityCheckReqs = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name:        metricNameSourceAvailabilityCheckRequestTotal,
+			Help:        "availability Check requests count partitioned by type (aws/gcp/azure), source status and error",
+			ConstLabels: prometheus.Labels{"service": "provisioning"},
```

Comment:
I suggest to also add another static label "component": "statuser" so we can tell which metrics are coming out from which components.